File: Java/Lista_04(Vetores)/Soma_Vetor.java
import java.util.Locale;
import java.util.Scanner;

public class Soma_Vetor {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner s = new Scanner(System.in);

        System.out.print("Quantos numeros voce vai digitar? ");
        int n = s.nextInt();

        double vet[] = new double[n];

        for (int i = 0; i < n; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = s.nextDouble();
        }

        System.out.print("\nVALORES: ");

        for (int i = 0; i < n; i++) {
            System.out.printf("%.2f ", vet[i]);
        }

        double soma = 0;

        for (int i = 0; i < n; i++) {
            soma += vet[i];
        }

        System.out.printf("\nSOMA = %.2f\n", soma);

        double media = soma / n;

        System.out.printf("MEDIA = %.2f", media);

        s.close();

    }
}

File: Python/Lista_04(Vetores)/Soma_Vetor.py
n = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(n)]

soma = 0.0

for i in range(0,n):
    vet[i] = float(input("Digite um numero: "))
    soma += vet[i]

media = soma/n

print("\nVALORES = ", end="")

for i in range(0,n):
    print(f"{vet[i]:.1f}", end=" ")

print(f"\nSOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")